prebid-mobile/modules/rendering/android-sdk-integration-gam.md

```diff
@@ -165,15 +153,14 @@ Pay attention that the `loadAd()` should be called on the main thread.
 
 The **default** ad format for an interstitial ad is **DISPLAY**. In order to make a `multiformat bid request`, set the respective values into the `adUnitFormats` parameter.
 
-```
+```kotlin
```

Comment:
It is not related to the changes in the PR, but I see inconsistency with the current code. The `InterstitialAdUnit` doesn't have such initialization properties as `minSizePercentage`. It is not in iOS or the Android SDK. It looks like a legacy code. 

We can create separate ticket to address legacy code issues discovered in this PR.

Reply:
The follow-up ticket